pmlc/target/x86/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ pml_cc_library(
 
 add_custom_command(
   OUTPUT heatmap.cc
-  COMMAND python
+  COMMAND ${PYTHON_EXECUTABLE}
```

Comment:
Why do we need this and where does `PYTHON_EXECUTABLE` come from?

Reply:
This was causing a build error for me because the command needed the python executable from `.cenv/` to run correctly due to the pystache requirement; on my build `python` was finding my machine's python instead which did not have pystache installed. 

I believe it gets set with `find_package(Python)` in `cmake/pml_format.cmake`